source/eventhandler.ts

```diff
@@ -250,6 +311,42 @@ export class EventHandler {
         this.pushMouseEventHandler(MouseEventProvider.Type.Wheel, handler);
     }
 
+    /**
+     * Register a touch start event handler that is to be called on update iff at least a single touch start event has
```

Comment:
Spelling: `iff`. Also a few times below.

Reply:
I used the wording which was also used for mouse events. I think it's used as in math: iff = "if and only if"